src/internal/components/ScrollableRegion.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react'
 import Box from '../../Box'
 import {useOverflow} from '../hooks/useOverflow'
+import {SxProp} from '../../sx'
 
 type ScrollableRegionProps = React.PropsWithChildren<{
   'aria-labelledby'?: string
   className?: string
-}>
+}> &
+  SxProp
```

Comment:
I think I'm missing something. Why do we need to start supporting `sx` on `ScrollableRegion`?

Reply:
Yes, because if we don't, the footers in the bottom sheets and full screens won't stay at the bottom if the content isn't long enough. That's why I'm using this method:

`<ScrollableRegion aria-labelledby={dialogLabelId} className="DialogOverflowWrapper" sx={{flex: 1}}>`